projects/core/src/routing/protected-routes/protected-routes.service.ts

```diff
@@ -22,6 +22,13 @@ export class ProtectedRoutesService {
     }
   }
 
+  /**
+   * Tells if the app is protected
+   */
+  isAppProtected(): boolean {
```

Comment:
Is it really needed? What do you thing about simply making `shouldProtect` public?

Reply:
I'm ok with making `shouldProtect` public I guess. But on the other hand, I would like to hide it behind a method, so we have this one layer, in case we change `shouldProtect` to something else. so only the method would be public, but the implementation would be hidden and protected/private - future proof solution